sbt-play-ebean/src/main/scala/play/ebean/sbt/PlayEbean.scala

```diff
@@ -25,7 +40,24 @@ object PlayEbean extends AutoPlugin {
 
   override def trigger = noTrigger
 
-  override def projectSettings = inConfig(Compile)(scopedSettings) ++ unscopedSettings
+  override def projectSettings = {
+    val querySettings = Seq(
+      playEbeanQueryGenerate := false,
+      playEbeanQueryEnhance := false,
+      playEbeanQueryDestDirectory := "app",
```

Comment:
"Query Beans" are not expected to be modified (ever). 

The generator also can optionally generate "Finders" like "CustomerFinder" and these are intended to me modified and I'm sure that is what Andrew was meaning. These finders have a where() method that returns an appropriate query bean.  For example CustomerFinder has a byName() method added manually.
 https://github.com/ebean-orm/avaje-ebeanorm-examples/blob/master/a-basic/src/main/java/org/example/domain/finder/CustomerFinder.java